### PR TITLE
Ros2 devel resume

### PR DESCRIPTION
**Pull request by Pugens**
First steps towards bringing into our main repo the past work from @dave992 @rtonnaer @NithishkumarS and @christianlandgraf

- The CI is failing, as latest ros2 is the taget
- For now, only the description pakage is working as expected, with improvements needed yet
- Dev setup can be shared on this branch as well if needed, but main points to get it to work are:
   - Changes done by @NithishkumarS [here](https://github.com/sam-xl/scancontrol/pull/34), namely move to scanCONTROL SDK 1.0.0 and Aravis 0.8
   - Suggestions by @erichlf to compile the SDK on Ubuntu 22, namely, the lltlib header file needs changing [here](https://github.com/Pugens/scancontrol/blob/1105de0ea8a28526b03de488d76821e07bada265/micro_epsilon_scancontrol_driver/include/lltlib/llt.h#L27-L33)

Cheers!

### CODE REVIEW

File: .docker/Dockerfile
Comment (gavanderhoorn) [inline .docker/Dockerfile:1]: high-level, first impression, comment: have we considered splitting this? Seems to build a nr of things, might be worthwhile to build the two dependency `.deb`s in separate images, then use the results to build the actual driver image?

Comment (Pugens) [reply .docker/Dockerfile:1]: I am not focusing on CI right now. This is mostly a placeholder PR. 
I'd rather worry about having a successful CI for jammy right now, if you could help pls?
I can share the Dockerfile that works for my dev env, from there it should be esasy to optimize to CI

**Comment by Pugens** [comment]
Added temporary Dockerfile image on my personal dockerhub for CI

**Comment by Pugens** [comment]
Added LIC Apache2.0 properly

**Comment by Pugens** [comment]
@gavanderhoorn I know it's a big PR, but it is all WIP, can anyone accept it now so I can do more granular updates?
Otherwise the diff will just increase.

Next steps. I will open separate issues for each, if we get this PR done pls:
- [ ] Check intensity integration from @christianlandgraf 
- [ ] Fix tfs as the profile is not showing on rviz2 beacuse of some issues related to them
- [ ] Improve reliability at connection/disconnection to avoid possible false/negligible error `-1005` at connection (update firmware needed)
- [ ] etc.. etc..

**Pull request merged**